backends/bmv2/jsonconverter.cpp

```diff
@@ -1844,11 +1849,13 @@ void JsonConverter::addMetaInformation() {
   auto meta = new Util::JsonObject();
 
   static constexpr int version_major = 2;
-  static constexpr int version_minor = 4;
+  static constexpr int version_minor = 6;
   auto version = mkArrayField(meta, "version");
   version->append(version_major);
   version->append(version_minor);
 
+  meta->emplace("compiler", "https://github.com/p4lang/p4c");
```

Comment:
There are multiple back-ends with this url, maybe this should be p4c-bm2-ss instead.